### ISSUE TITLE
Does this lift myself up by the Boilerstraps?

### ISSUE THREAD

**Issue by marcus-at-localhost** [impressivewebs/HTML9-Responsive-Boilerstrap-js#174]
I was able to lift myself up by the bootstraps, but I can't figure out how to do that with the Boilerstraps. Any advice?

**Comment by bkb3** [comment]
That would be extremely painful.

**Comment by CrossEye** [comment]
Yes, please don't try this at home (or in any of the neighboring star systems either.)  Honestly, if you're going to do this, please try another galaxy.  Preferably one you don't care all that much about.
